apps/sms/services/js/moz_mobile_message/moz_mobile_message_shim.js

```diff
@@ -0,0 +1,210 @@
+/* -*- Mode: js; tab-width: 2; indent-tabs-mode: nil; c-basic-offset: 2 -*- /
+/* vim: set shiftwidth=2 tabstop=2 autoindent cindent expandtab: */
+
+
+/* global bridge, Utils */
+
+/* exported MozMobileMessageShim */
+
+'use strict';
+(function(exports) {
+
+/**
```

Comment:
Then all agreed - no indent here anymore :) Will have cleaner git-blame when file is migrated to es6 module.

Updated https://etherpad.mozilla.org/sms-code-style-guidelines